tools/commit-checker/check.py

```diff
@@ -486,7 +547,6 @@ def run_section(section, out) -> int:
         elif section == CLOSED_ILLEGAL_RESOLUTION:
             print("Tip: Fixed tickets should have resolution 'Fixed by Other Ticket' or 'Fixed'.", file=out)
```

Comment:
This doesn't seem right to me? There shouldn't be any commits against tickets with resolution 'Fixed by Other Ticker', so it's not clear to me why this is being recommended at the same time as the resolution 'Fixed'.

Reply:
it's kind of an ICU tip, I'm wondering about just removing this tip entirely because the new explanations will be much more on point.

Reply:
Up to you :)